lib/fs.js

```diff
@@ -385,6 +386,11 @@ fs.readFile = function(path, options, callback) {
   if (!nullCheck(path, callback))
     return;
 
+  //doing justice between linux and windows for `/` char at the end of file path
```

Comment:
space after comment please

Reply:
Space added thanks.

Reply:
I think what @devsnek meant was a space after the comment indicator (`//`) so: `// doing justice...` 

There are some lint errors you're going to want to correct (mostly around line lengths). These should have been flagged if you ran `make test` or `vcbuild test`. To just run the JS linting (since running the full tests can be time-consuming), you can use `make lint-js` or `vcbuild lint-js`.

Lastly, welcome @timotew and thanks for the pull request!

Reply:
@devsnek  sorry I misunderstood,  @Trott  Thanks a lot for having me and time for this.